Control/Monad/Logger.hs

```diff
@@ -883,8 +884,8 @@ logCS cs src lvl msg =
   monadLoggerLog (locFromCS cs) src lvl msg
 
 -- | Logs a message with location given by 'CallStack'.
--- See 'Control.Monad.Logger.CallStack' for more convenient
--- functions for 'CallStack' based logging.
+-- See "Control.Monad.Logger.CallStack" for more convenient
+-- functions for @CallStack@ based logging.
```

Comment:
What's the purpose of this change?

Reply:
This haddock currently tries to quote the `CallStack` module, but the link just leads back to `Control.Monad.Logger`. According to the haddock [documentation](https://haskell-haddock.readthedocs.io/en/latest/markup.html#linking-to-modules) you should quote modules with double quotes.

Reply:
That's fine on line 887. I'm asking about line 888: why did you change it from `'CallStack'` to `@CallStack@`?

Reply:
That isn't really a strict "improvement" of course. I merely thought it looked awkward that `CallStack` is quoted three times in a row in a single haddock, the third quotation being redundant because of the first one.

This really should've gone into a more general "doc improvements" patch. For instance the haddocks for `runWriterLoggingT` and `execWriterLoggingT` haddocks have issues as well.
I spent some time yesterday trying to figure out if this review section would become messed up if I ammended/rebased the tip of my fork. I now believe I should do so, and do everything not related to exporting the `LogLine` synonym in a feature-branch. I just wanted to get some confirmation first if that's how it should be done.

Sorry for the beginner mistakes.

Reply:
Yes, rebasing would be a good idea.